src/Graphics/Canvas.purs

```diff
@@ -121,10 +118,7 @@ foreign import data CanvasElement :: *
 foreign import data Context2D :: *
 
 -- | An image data object, used to store raster data outside the canvas.
-foreign import data ImageData :: *
-
--- | An array of pixel data.
-foreign import data CanvasPixelArray :: *
+type ImageData = { width :: Number, height :: Number, data :: Uint8ClampedArray }
```

Comment:
Yep, it's just a number. They're syntactically forced to be actual ints in the compiler, and the operators are modified to preserve integerness.